deeppavlov/models/ranking/matching_models/matching_predictor.py

```diff
@@ -0,0 +1,122 @@
+# Copyright 2018 Neural Networks and Deep Learning lab, MIPT
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+from typing import List, Iterable
+
+from deeppavlov.core.common.log import get_logger
+from deeppavlov.core.models.component import Component
+from deeppavlov.models.ranking.matching_models.tf_base_matching_model import TensorflowBaseMatchingModel
+from deeppavlov.core.common.registry import register
+
+log = get_logger(__name__)
+
+
+@register('matching_predictor')
+class MatchingPredictor(Component):
```

Comment:
Why is necessity for this class? It is very similar to `SiamesePredictor` class.